crates/futures/tests/tests.rs

```diff
@@ -155,3 +155,15 @@ async fn can_use_an_async_iterable_as_stream() {
     assert_eq!(stream.next().await, Some(Ok(JsValue::from(24))));
     assert_eq!(stream.next().await, None);
 }
+
+#[wasm_bindgen_test]
+#[should_panic]
+async fn should_panic() {
+    panic!()
+}
+
+#[wasm_bindgen_test]
+#[should_panic = "error message"]
```

Comment:
Is this syntax same as the one provided by stdlib? I don't remember the stdlib one. We need to ensure we don't introduce our own syntax here

Reply:
No, this is definitely not the same a what stdlib provides, a mistake on my part.
The parser is implemented correctly, so the message was simply ignored, as you mentioned before, a failing test would have caught that.

Reply:
Fixed.

Reply:
The stdlib actually supports both `#[should_panic = "..."]` and `#[should_panic(expected = "...")]`, so we should probably support both as well.

Reply:
Indeed, https://doc.rust-lang.org/reference/attributes/testing.html:
> The string may be passed using the [MetaNameValueStr](https://doc.rust-lang.org/reference/attributes.html#meta-item-attribute-syntax) syntax or the [MetaListNameValueStr](https://doc.rust-lang.org/reference/attributes.html#meta-item-attribute-syntax) syntax with an `expected` field.

Reply:
Done.